pom.xml

```diff
@@ -507,18 +507,35 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
-                        <configuration>
-                            <argLine>
-                                ${argLine}
-                                -Dhazelcast.test.defaultTestTimeoutInSeconds=600
-                            </argLine>
-                            <forkCount combine.self="override">1</forkCount>
-                            <groups combine.self="override">
-                                com.hazelcast.test.annotation.NightlyTest,
-                                com.hazelcast.test.annotation.SlowTest
-                            </groups>
-                            <excludedGroups combine.self="override" />
-                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>serial-tests</id>
```

Comment:
what about ` ${argLine}` here?

Reply:
We're skipping whole execution for id `serial-tests` in nightly profile.